src/app.module.ts

```diff
@@ -15,7 +18,7 @@ import { User } from './users/model/user.model';
       username: process.env.DB_USERNAME,
       password: process.env.DB_PASSWORD,
       database: process.env.DB_DATABASE,
-      models: [User],
+      models: [User, Group],
       autoLoadModels: true,
```

Comment:
```suggestion
      autoLoadModels: true,
```
you don't need to specify your models when you using `autoLoadModels: true` property